src/concrete/ml/torch/compile.py

```diff
@@ -142,6 +143,8 @@ def _compile_torch_or_onnx_model(
         global_p_error (Optional[float]): probability of error of the full circuit. In FHE
             simulation `global_p_error` is set to 0
         verbose (bool): whether to show compilation information
+        inputs_encryption_status (Optional[Sequence[str]]): encryption status ('clear', 'encrypted')
```

Comment:
same

Reply:
Same what? There is a comment on the fact that by default everything is encrypted

Reply:
same as in "same comment as above" in order to not avoid making changes in both places if there are some